### DISCUSSION TITLE
Map with differents layers having differents CRS

### DISCUSSION THREAD

**Discussion by edfr73** [Mapsui/Mapsui#1793] [Q&A]
Hello, 

I write an Android App using differents map layers : OpenTopoMap, OpenStreetMap, Google Satellite, and fench "Cartes IGN" with EPSG:3857 and I try to use french "SCAN25 Touristique" with EPSG:2154.

The problem occurs with "SCAN25 Touristique", it is well displayed on the map but not at the right place. So, my question is : Is it possible to have this kind of configuration with Mapsui ? 

I use MapSui 4.0.0-beta.5 and BruTiles 5.0.3 to have an access to IGN WMTS service.

My code to initiate map is like this, as I want the map to be displayed in EPSG:3857 : 

var map = new Mapsui.Map
{
   CRS = "EPSG:3857",
};
map.Layers.Add(OpenTopoMap.CreateTileLayer());
map.Layers.Add(OpenStreetMap.CreateTileLayer());
...

**Comment by pauldendulk** [comment]
In Mapsui it is not possible to do transformation of raster data (like the WMTS tiles) from one coordinate system to another. It is possible to implement something like this but the result usually looks ugly and not fast. SharpMap had it implemented so we might also add this at some point, but we have a lot on our backlog.

A WMTS service can serve several tile sets. Then often have the same data in different projections. Perhaps you can check if SCAN25 Touristique also serves a set in EPSG:3857.

**Reply by edfr73** [reply]
Thanks for your answer, that is what I thougth but I was not sure. Finaly I found an other tile set with nearly the same data in EPSG:3857, my beta tester said that it is ok for him 😀
